### ISSUE TITLE
Phase 2 (Stage 1): Provision H100 Node & Deploy Lightning Environment

### ISSUE THREAD

**Issue by nochadisfaction** [pixelatedempathy/pixelated#541]
# Objective

Set up the remote computing infrastructure necessary to train a 12.3 Billion parameter model utilizing DeepSpeed mixed-precision training. The configuration must be reproducible and isolated.

## Hardware Requirements

* **Primary Preference**: 1x H100 (80GB VRAM) or 2x A100 (80GB VRAM) compute nodes.
* **System Memory**: 64GB+ Host RAM.
* **Persistent Block Storage**: Minimum 100GB to manage parameter weight checkpoints and intermediate `.safetensors`.

## Deployment Playbook (VPS Node actions)

1. **Environment Setup**<br>Clone the codebase and initiate package resolution strictly utilizing `uv`:

```bash
git clone <pixelated-repo-url> ~/pixelated
cd ~/pixelated
pnpm install
cd ai && uv sync
```

2. **Inject Credentials**<br>Mount the required environment secrets mapped to OVH and W&B:

```bash
export OVH_S3_BUCKET=pixel-data
export OVH_S3_ENDPOINT=https://s3.us-east-va.io.cloud.ovh.us
export OVH_S3_ACCESS_KEY=<redacted>
export OVH_S3_SECRET_KEY=<redacted>
export WANDB_API_KEY=<redacted>
```

3. **PyTorch Validation**<br>Execute the dry-run script locally on the node to ensure the C++ extensions for Flash Attention and bitsandbytes initialize against the GPU fabric:

```bash
uv run python ai/lightning/production/train_therapeutic_ai.py --stage 1 --dry-run
```

## Acceptance Criteria

- [ ] Provider secured and compute node active via SSH.
- [ ] Python ecosystem isolated using `uv` virtual environment mapping to PyTorch Lightning 2.4+.
- [ ] S3 loaders can reach `pixel-data` natively avoiding rate limiting.
- [ ] The `train_therapeutic_ai.py` `--dry-run` successfully triggers a batch pass and reports validation loss metrics without OOMing or throwing meta tensor errors.
